components/Paragraph.js

```diff
@@ -0,0 +1,13 @@
+import { BodyComponent } from 'mjml-core'
+
+export default class Paragraph extends BodyComponent {
+  render() {
+    return this.renderMJML(`
+     <mj-section>
+        <mj-column>
+          <text>${this.getContent()}</text>
+        </mj-column>
+      </mj-section>
+    `)
+  }
+}
```

Comment:
Yeah that's what I though, as `css-class` is inherited from mj-column you would need to apply your css-classs to `<text>` here as it won't be available on `<text>` render function.

There's nothing you can do about this :/

Reply:
thanks :-(